src/ensembl/src/shared/components/question-button/QuestionButton.test.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import QuestionButton from './QuestionButton';
+import { TOOLTIP_TIMEOUT } from 'src/shared/components/tooltip/tooltip-constants';
+
+describe('<QuestionButton />', () => {
+  const helpMessage = 'I am a helpful message that will appear in the tooltip';
+  const helpText = <span>{helpMessage}</span>;
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  describe('on hover', () => {
+    it('shows the tooltip after a delay', async () => {
+      const { container, queryByText } = render(
+        <QuestionButton helpText={helpText} />
+      );
+      const questionButton = container.querySelector(
+        '.questionButton'
+      ) as HTMLElement;
+
+      userEvent.hover(questionButton);
+      expect(questionButton.querySelector('.pointerBox')).toBeFalsy();
+
+      act(() => {
+        jest.advanceTimersByTime(TOOLTIP_TIMEOUT + 10);
```

Comment:
TOOLTIP_TIMEOUT = 800  Is it in milliseconds? If so, does it wait for 8 seconds here when we run this test?